alpine/alpine-server/src/main/java/alpine/server/filters/AuthenticationFilter.java

```diff
@@ -79,8 +80,7 @@ public void filter(ContainerRequestContext requestContext) {
                     }
                 } catch (AuthenticationException e) {
                     LOGGER.info(SecurityMarkers.SECURITY_FAILURE, "Invalid API key asserted");
-                    requestContext.abortWith(Response.status(Response.Status.UNAUTHORIZED).build());
-                    return;
+                    throw new NotAuthorizedException(Response.status(Response.Status.UNAUTHORIZED).build());
```

Comment:
Necessary in order for `ExceptionMapper`s to be informed about authentication failures.